### PR TITLE
Descendant Reverse Fix

### PR DESCRIPTION
**Pull request by alongreyber**
Added a test for a case where the query graph includes related_to and we need to subclass related_to in both forward and reverse directions. Implemented this change by switching query graph permutations to operation graph permutations.

### CODE REVIEW

File: tests/query_planner/ex1_qg.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-      "object" : "n0",
-      "predicate" : "biolink:treated_by"
-    },
-    "e02" : {
-      "subject" : "n0",
-      "object" : "n2",
-      "predicate" : "biolink:affects_abundance_of"
-    }
-  }
+	"nodes": {
```
Comment (patrickkwang) [inline tests/query_planner/ex1_qg.json:2]: Are these tabs?

File: tests/query_planner/test_query_planner.py [hunk 2]
```diff
@@ -412,7 +421,8 @@ async def test_unbound_unconnected_node(caplog):
         n2(( category biolink:PhenotypicFeature ))
         """
     )
-    qg = await expand_qg(qg, logging.getLogger())
+    await fill_categories_predicates(qg, logging.getLogger())
```
Comment (patrickkwang) [inline tests/query_planner/test_query_planner.py:424]: We seem to do these together a lot. Should we make a wrapper for these two functions?

File: tests/query_planner/ex1_qg.json (hunk 1 above)
Comment (alongreyber) [reply tests/query_planner/ex1_qg.json:2]: Not as far as I can tell

Comment (alongreyber) [reply tests/query_planner/ex1_qg.json:2]: Okay they actually are tabs - I'm working on a fix

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**